pkg/proto/configuration/bb_worker/bb_worker.proto

```diff
@@ -110,6 +111,15 @@ message NativeBuildDirectoryConfiguration {
   // LEAST_RECENTLY_USED can, assuming the cache size is sufficient.
   buildbarn.configuration.eviction.CacheReplacementPolicy
       cache_replacement_policy = 5;
+
+  NativeBuildDirectoryLinkType link_type = 6;
```

Comment:
I'm fine with just using a boolean here: `use_clonefile`. We can always extend it to have a `oneof` by the time we add even more mechanisms ;-)

Reply:
Let's just add that functionality when we get there. There's no harm in changing the schema, especially for features that were recently added.

Reply:
Well, yes, but for the local code I have for followup that does